scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -291,7 +291,7 @@ <h5 class="modal-title" id="exampleModalLabel">Beacon submission</h5>
 
 {% macro pretty_variant(variant) %}
 
-  {% if variant.category in "str" %}
+  {% if variant.category and variant.category in "str" %}
```

Comment:
Good, but either on all the ifs in the block, or in a surrounding if, right? You would only go to the elif if category is not in variant?

Reply:
Good point. I just added this when one of the variants pages crashed - after I removed all these vars in the category I guess